members/migrations/0003_auto_20201227_1351.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.15 on 2020-12-27 18:51
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+import members.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('auth', '0008_alter_user_username_max_length'),
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('members', '0002_auto_20201221_2031'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='CommitteeMembership',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('start', models.DateField(default=django.utils.timezone.localdate)),
+                ('stop', models.DateField(blank=True, null=True)),
+                ('committee', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='users', to='auth.Group')),
+            ],
+            options={
+                'abstract': False,
+            },
+        ),
+        migrations.CreateModel(
+            name='Position',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=50)),
+                ('index', models.SmallIntegerField(default=0, help_text='Order to render on page.')),
```

Comment:
should this be a PositiveSmallIntegerField (like Hackaton?) I was able to set index equal to -1

Reply:
see iquhack/models.py line 192